apps/charterafrica/src/components/Tool/ContributeAndGoToRepo.js

```diff
@@ -0,0 +1,84 @@
+import { Link } from "@commons-ui/next";
+import { Box, Button, SvgIcon } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+import ContributeIcon from "@/charterafrica/assets/icons/Type=contribute, Size=24, Color=CurrentColor.svg";
+import GithubIcon from "@/charterafrica/assets/icons/Type=github, Size=24, Color=CurrentColor.svg";
+
+const ContributeAndGoToRepo = React.forwardRef(function ContributeAndGoToRepo(
```

Comment:
Doesn't contribute link to repo?

Reply:
Go to repo should go to repository.
Contribute can redirect to issues

Reply:
Nah, I think Contribute was supposed to be something else... may be login and contribute to charter.africa rather than the tool. Similar to [this](https://ovio.org/project/vercel/next.js).

Confirm with the team but I think we don't need it.